### PR TITLE
Prepare release v1.0.0

### PR DESCRIPTION
**Pull request by okkez**
* Add changelog since the initial release.
* Bump version.

I've added changelogs except internal changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* Support tagging ECS resources
+  https://github.com/reproio/ecs_deploy/pull/48
+  https://github.com/reproio/ecs_deploy/pull/49
+* Wait for stopping tasks until tasks stop
+  https://github.com/reproio/ecs_deploy/pull/50
+* Improve performance when start tasks
+  https://github.com/reproio/ecs_deploy/pull/50
+* Improve stability of auto scaling groups managed by ecs_auto_scaler
+  https://github.com/reproio/ecs_deploy/pull/55
+
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion

### Bug fixes

* Fix infinite loop that occurs when there are more than 100 container instances
  https://github.com/reproio/ecs_deploy/pull/38
* Fix errors that occur on decreasing more than 20 container instances
  https://github.com/reproio/ecs_deploy/pull/39
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  https://github.com/reproio/ecs_deploy/pull/40
+* Add network_configuration paramters to ScheduledTask
+  https://github.com/reproio/ecs_deploy/pull/46
+* Support tagging ECS resources
+  https://github.com/reproio/ecs_deploy/pull/48
+  https://github.com/reproio/ecs_deploy/pull/49
+* Wait for stopping tasks until tasks stop
+  https://github.com/reproio/ecs_deploy/pull/50
+* Improve performance when start tasks
+  https://github.com/reproio/ecs_deploy/pull/50
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion
  https://github.com/reproio/ecs_deploy/pull/53
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+# Ancient releases
+
+## Release v0.3.2 - 2017/23/10
+
+### Enhancement
+
+* Remove execution feature
+  https://github.com/reproio/ecs_deploy/pull/24
+
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion
* Support container overrides in scheduled tasks
  https://github.com/reproio/ecs_deploy/pull/26

### Bug fixes

* Fix deployment errors that occur when `ecs_scheduled_tasks` is not set
  https://github.com/reproio/ecs_deploy/pull/27

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+### Enhancement
+
+* Add parameters `ecs_wait_until_services_stable_max_attempts` and `ecs_wait_until_services_stable_delay`
+  https://github.com/reproio/ecs_deploy/pull/30
+* Detect region automatically according to AWS SDK
+  https://github.com/reproio/ecs_deploy/pull/31
+* Support new features of ECS to add Fargate support
+  https://github.com/reproio/ecs_deploy/pull/32
+* Drop AWS SDK 2 support
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion
* Ignore running tasks which don't belong to the ECS services on deregistering container instances
  https://github.com/reproio/ecs_deploy/pull/33
* Drop AWS SDK 2 support
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+### Bug fixes
+
+* Fix block parameter name
+
+## Release v0.3.0 - 2017/03/08
+
+### Enhancement
+
+* Support network_mode and placement_constraints
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion
* Support network_mode and placement_constraints
* Introduce `ecs_registered_tasks` capistrano variable
  https://github.com/reproio/ecs_deploy/pull/23
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+* Add tasks to deploy the application faster
+  https://github.com/reproio/ecs_deploy/pull/57
+
+### Enhancement
+
+* Add parameters `ecs_wait_until_services_stable_max_attempts` and `ecs_wait_until_services_stable_delay`
+  https://github.com/reproio/ecs_deploy/pull/30
+* Detect region automatically according to AWS SDK
+  https://github.com/reproio/ecs_deploy/pull/31
+* Support new features of ECS to add Fargate support
```
Comment (abicky) [inline CHANGELOG.md]:
```suggestion
* Support new features of ECS to support Fargate
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+* Support network_mode and placement_constraints
+
+### Bug fixes
+
+* Filter inactive services
+  https://github.com/reproio/ecs_deploy/pull/19
+* Wait 10 services at once
+  https://github.com/reproio/ecs_deploy/pull/20
+  https://github.com/reproio/ecs_deploy/pull/21
+* Support ScheduledTask deployment
```
[the remark this answers is not in the document]
Comment (okkez) [reply CHANGELOG.md]: Fixed 6da0b31

**Comment by okkez** [comment]
Released v1.0.0
https://rubygems.org/gems/ecs_deploy/versions/1.0.0

**Pull request merged**